_posts/2019/2019-05-04-react-hooks-what-all-the-fuss-is-about.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: React Hooks - What all the fuss is about
```

Comment:
(i) This title ranks at 66 points. (https://coschedule.com/headline-analyzer#)

Reply:
I take that.